src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -2865,7 +2886,7 @@ public Long zcount(final byte[] key, final byte[] min, final byte[] max) {
 
   protected Set<Tuple> getTupledSet() {
     List<byte[]> membersWithScores = client.getBinaryMultiBulkReply();
-    if (membersWithScores.isEmpty()) {
+    if (membersWithScores == null || membersWithScores.isEmpty()) {
```

Comment:
For which case are you getting `membersWithScores == null`?

Reply:
When the key was not exist, it will throw NPE.

Reply:
return `null` in that particular case.